tools/system_libs.py

```diff
@@ -1843,6 +1843,14 @@ def get_default_variation(cls, **kwargs):
     )
 
 
+class libwebgpu(MTLibrary):
+  name = 'libwebgpu'
+
+  cflags = ['-std=c++20']
```

Comment:
Is it possible this could cause issues if folks are linking this library with c++17 (the default) or c++11?

Reply:
I don't think so. The linkage should always be plain C since it's only through the C header.
C++17 would almost certainly be fine though if that's the default (I wasn't sure what was best to put here)

Reply:
I was thinking about the libc++ library symbols that libgpu could end up depending on.. but thinking about it that libraries doesn't vary with `-std` version.. so it should be fine (i.e. its ok to link `-std=c++20` and `-std=c++11` code in the same binary I think).

Reply:
If you are OK with with C++17 for now I would remove this line then.